pandas/tests/indexing/test_loc.py

```diff
@@ -1191,7 +1191,9 @@ def test_loc_setitem_multiindex_slice(self):
     def test_loc_getitem_slice_datetime_objs_with_datetimeindex(self):
         times = date_range("2000-01-01", freq="10min", periods=100000)
         ser = Series(range(100000), times)
-        ser.loc[datetime(1900, 1, 1) : datetime(2100, 1, 1)]
+        result = ser.loc[datetime(1900, 1, 1) : datetime(2100, 1, 1)]
+        expected = Series(range(100000), times)
```

Comment:
cant you  just use `expected = ser`?

Reply:
Thx, changed it. Was not sure if it would be better to redefine the expected value